README.md

```diff
@@ -47,7 +47,7 @@ echo "'use strict'; function hello() { print('Hello World'); } hello();" | ./bin
 
 ## Contributing
 
-The main purpose of this repository is to continue to evolve Hermes, making it faster and more efficient. We are grateful to the community for contributing bugfixes and improvements. Read below to learn how you can take part in improving Hermes.
+The main purpose of this repository is to continue to evolve Hermes, making it faster and more efficient. We are grateful to the community for contributing bug fixes and improvements. Read below to learn how you can take part in improving Hermes.
```

Comment:
I should be bugfix (or bug-fix), shouldn't it? (https://www.quora.com/Is-the-word-bugfix-or-bug-fix)